### DISCUSSION TITLE
Able to completele disable enable-disable mesh renderers, skined mesh renderers, etc on nobs

### DISCUSSION THREAD

**Discussion by SnaiperoG3D** [FirstGearGames/FishNet#325] [Feature Request]
There are many situations when this feature confuse,
For example i have a lot of models on player, i enable and disable it myself, but nob disable by itself, even on client.
Maybe its releated to network pool somehow, cuz i have a pre spawn feature. Also i cant debug problem places, for example i have a bug when one player is invisible for other players, but not all of them, some one can cleary see him. I guess its related to pool and this disable-enable mrs and smrs feature. Good to have to completele disable it on nobs. Also good to have it for more space, cuz i want to controll it by myself

**Comment by FirstGearGames** [comment]
I want to accept this but I'm not going to put all of that work onto FishNet. Let's think of a good way to let the user implement this themselves. An extendible or scriptable way would be great, and I can set the default entry to how FishNet currently works with just renderers.

**Comment by SnaiperoG3D** [comment]
hm, am i found right file, is it controlled via NetworkObject.Observers.cs? I did some changes to this file, but not tested yet in proper way. in my case i did just simple toggle to disable it, but we can come up with something here

**Comment by FirstGearGames** [comment]
Added in 3.7.8
- Added NetworkObject.OnHostVisibilityUpdated.
